### DISCUSSION TITLE
For q-carousel within nested v-for loop, how to assign the v-model with the name of the first slide?

### DISCUSSION THREAD

**Discussion by dost-mmg** [quasarframework/quasar#12495] [General - Components / Directives / etc]
The problem is that the first slide is blank, but everything else works. I think the issue is that the v-model does not refer to the name of the first slide. How do I reference the name of that first slide?

Note: the locations array contains locImgPaths, locNumb, locName values. The locImgPaths is an array of all img urls I want to display for the particular location, containing filepath and url values.

There are many resources that come close to showing me the light, including this one: https://forum.quasar-framework.org/topic/3162/qcarousel-slide-not-showing-the-first-slide/4.

```
<template>
   <div v-for="(loc, i) in locations" :key="i">
      <q-carousel
          v-model="slides[i]" <--this needs to be the name of the first slide
       >
           <q-carousel-slide
               v-for="(img, id) in loc.imgUrls"
               :key="id"
               :name="id"
               :img-src="img.url"
             />
        </q-carousel>
    </div>
    </template>
     <script>
          export default {
              setup() {
                let locations = ref([]);
                let slides = ref({});
            return {
                 locations,
                 slides,
             };
          },
       };
< /script>
```

**Comment by pikil** [comment]
@kiggs1881 Is is possible to create a codepen with the problem so forum participants can have a look?

**Reply by dost-mmg** [reply]
Working on it now. I'm getting an object error, though: https://codepen.io/kiggs1881/pen/gOXXNqG (first time trying). Will update when I've figured it out.

**Reply by pikil** [reply]
Just take any codepen example from any Quasar docs page provided by Q-team - these folks have injected all the necessary dependencies, so you can just have a shortcut there :)
